### DISCUSSION TITLE
Add Alpine code to UI examples.

### DISCUSSION THREAD

**Discussion by bastinald** [tailwindlabs/tailwindcss#5323] [Ideas]
As someone who uses Livewire + Alpine, the React and Vue examples are of little use to me. I purchased the UI package and am happy with it, but it would be nice if we had access to the working code actually used in the HTML preview.

I understand you have not made this available because of concerns with accessbility and possible bugs, so you could offer a disclaimer such as `this code may contain bugs or accessibility issues, use at own risk`. I have used dev tools to copy the code with next to no issues, however. While using dev tools to copy the code works, it is a huge timesink and quite frankly, a pain in the behind.

Right now, the only way to do this more efficiently is to use a TamperMonkey extension such as https://gist.github.com/KevinBatdorf/8bd5f808fff6a59e100dfa08a7431822

It would be nice if we could just copy and paste the HTML + Alpine code. 

Thanks for your consideration.

**Comment by dlabsnl** [comment]
It's been 3,5 years. I don't see any replies, only 3 users with a up-vote.

I agree with this. I would like to see Livewire next to Vue and React as a example code.
